website/showcase.json

```diff
@@ -70,6 +70,38 @@
       "pinned": true
     }
   ],
+  "amazon": [
+    {
+      "name": "Amazon Shopping",
+      "icon": "amazon-shopping.png",
+      "linkAppStore": "https://apps.apple.com/us/app/amazon-shopping/id297606951",
+      "linkPlayStore": "https://play.google.com/store/apps/details?id=com.amazon.mShop.android.shopping"
+    },
+    {
+      "name": "Amazon Alexa",
+      "icon": "amazon-alexa.webp",
+      "linkAppStore": "https://apps.apple.com/us/app/amazon-alexa/id944011620",
+      "linkPlayStore": "https://play.google.com/store/apps/details?id=com.amazon.dee.app&hl=en_US&gl=US"
+    },
+    {
+      "name": "Amazon Photos",
+      "icon": "amazon-photos.webp",
+      "linkAppStore": "https://apps.apple.com/us/app/amazon-photos-photo-video/id621574163",
+      "linkPlayStore": "https://play.google.com/store/apps/details?id=com.amazon.clouddrive.photos&hl=en_US&gl=US "
+    },
+    {
+      "name": "Amazon Kindle",
+      "icon": "amazon-kindle.png",
+      "infoLink": "https://www.amazon.com/b/?node=6669702011",
```

Comment:
Is there a reason that there isn't a link to the [App](https://apps.apple.com/us/app/amazon-kindle/id302584613)/[Play](https://play.google.com/store/apps/details?id=com.amazon.kindle&hl=en_US&gl=US) store for the Kindle app?

Reply:
@mattcarrollcode -- yeah so the answer is a bit in the weeds here but after talking with the teams I think it will make sense:
1.. The mobile apps for iOS and Android are _not_ React Native, those ones are native-built by different eng teams.
2. The core Kindle experience **on the physical Kindle devices** is powered by React Native 🥳  

I know it's a bit different from the other common Android & Apple use cases on the showcase page but it's a unique use case worth highlighting for developers _(and validating for the versatility of React Native!)_

Reply:
1. Gotcha, makes sense
2. Woah, very cool!